pipeline-scripts/release.groovy

```diff
@@ -460,4 +460,229 @@ def signArtifacts(Map signingParams) {
+                            }
+                            // For non-candidate, put a hold on the PR to prevent accidental merging
+                            labelArgs = "-l 'do-not-merge/hold'"
+                            break
+                        case 'stable':
+                            // For non-candidate, put a hold on the PR to prevent accidental merging
+                            labelArgs = "-l 'do-not-merge/hold'"
+                            pr_messages << "Please merge within 48 hours of ${errata_url} shipping live OR a Cincinnati-first release."
+
+                            if (prURLs.containsKey('prerelease')) {
+                                pr_messages << "This should provide adequate soak time for prerelease channel PR ${prURLs.prerelease}"
+                            }
+                            if (prURLs.containsKey('fast')) {
+                                pr_messages << "This should provide adequate soak time for fast channel PR ${prURLs.fast}"
+                            }
+
+                            break
+                        default:
+                            error("Unknown prefix: ${prefix}")
+                    }
+
+                    if ( addToUpgradeChannel ) {
+                        pr_messages << "This PR will also enable upgrades from ${releaseName} to releases in ${upgradeChannel}"
+                    }
+
+                    if ( !addToChannel && !addToUpgradeChannel ) {
+                        def pr_info = "No Cincinnati PRs opened for ${prefix}. Might have been done by previous architecture's release build.\n"
+                        echo pr_info
+                        currentBuild.description += pr_info
+                        continue
+                    }
+
+                    withCredentials([string(credentialsId: 'openshift-bot-token', variable: 'access_token')]) {
+                        def messageArgs = ''
+                        for ( String msg : pr_messages ) {
+                            messageArgs += "--message '${msg}' "
+                        }
+
+                        sh """
+                            set -euo pipefail
+                            set -o xtrace
+                            if git checkout origin/${branchName} ; then
+                                echo "The branch ${branchName} already exists in cincinnati-graph-data. No additional PRs will be created."
+                                exit 0
+                            fi
+                            git branch -f ${branchName} origin/master
+                            git checkout ${branchName}
+                            if [[ "${addToChannel}" == "true" ]]; then
+                                echo >> ${channelFile}    # add newline
+                                echo '# Errata: ${errata_url}' >> ${channelFile}    # add link to errata for reference
```

Comment:
to be crystal clear which release the comment refers to, how about:
```suggestion
                                echo '# ${releaseName} Errata: ${errata_url}' >> ${channelFile}    # add link to errata for reference
```

Reply:
Next push